crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -613,25 +613,17 @@ fn check_name(checker: &Checker, expr: &Expr, range: TextRange) {
         }
 
         // airflow.datasets
-        ["airflow", "Dataset"] => Replacement::Name("airflow.sdk.definitions.asset.Asset"),
+        ["airflow", "Dataset"] => Replacement::Name("airflow.sdk.Asset"),
         ["airflow", "datasets", "DatasetAliasEvent"] => Replacement::None,
-        ["airflow", "datasets", "Dataset"] => {
-            Replacement::Name("airflow.sdk.definitions.asset.Asset")
-        }
-        ["airflow", "datasets", "DatasetAlias"] => {
-            Replacement::Name("airflow.sdk.definitions.asset.AssetAlias")
-        }
-        ["airflow", "datasets", "DatasetAll"] => {
-            Replacement::Name("airflow.sdk.definitions.asset.AssetAll")
-        }
-        ["airflow", "datasets", "DatasetAny"] => {
-            Replacement::Name("airflow.sdk.definitions.asset.AssetAny")
-        }
+        ["airflow", "datasets", "Dataset"] => Replacement::Name("airflow.sdk.Asset"),
```

Comment:
We can combine these two to avoid the `Replacement` duplication:

```rs
["airflow", "Dataset"] | ["airflow", "datasets", "Dataset"] => ...
```

Reply:
Done